### DISCUSSION TITLE
Please give basic examples in the documentation

### DISCUSSION THREAD

**Discussion by nicolasmbronner** [SBoudrias/Inquirer.js#1410] [General]
Like everything in life, nothing better than working examples to get people familiar with the tool 😊

With the followed code in the documentation, i couldn't make anything work on my own :
```
import inquirer from 'inquirer';

inquirer
  .prompt([
    /* Pass your questions in here */
  ])
  .then((answers) => {
    // Use user feedback for... whatever!!
  })
  .catch((error) => {
    if (error.isTtyError) {
      // Prompt couldn't be rendered in the current environment
    } else {
      // Something else went wrong
    }
  });
```

And after making some more research online, i found this example, which is perfect to introduce the package : 
```
const inquirer = require('inquirer');

inquirer
  .prompt([
    {
      name: 'faveReptile',
      message: 'What is your favorite reptile?'
    },
  ])
  .then(answers => {
    console.info('Answer:', answers.faveReptile);
  });
```

Atleast here we have something working, and we can dig it up with the rest of the documentation 🙏

**Comment by SBoudrias** [comment]
Hi, thanks for the feedback. Did you try the new packages and the new documentation?

The previous `inquirer` package is also overly complex; which makes this more of an issue.

**Reply by nicolasmbronner** [reply]
Thanks for your reply! Unless the latest version have a different package name, yes, i'm looking at the documentation of the v.9.2.23 which was published 4 days ago, from what i can read. Hmmm, am i missing something? I'll just paste here the link to the package i'm looking at : [Link](https://www.npmjs.com/package/inquirer).

**Reply by SBoudrias** [reply]
Ohh, the new package is `@inquirer/prompts`. I deployed under a different name to not break backward compatibility. The readme of the github repo reflect the new package(s): https://github.com/SBoudrias/Inquirer.js

**Reply by nicolasmbronner** [reply]
Great, thanks a lot for taking the time to answer, and in the new package, looks much better documented! 🙌
